Feature/advance docs (#35)
infinum/eightshift-docs: docs/advanced/helpers-object.md

---
id: helpers-object-helpers
title: Object
---

> Object trait is located in `Eightshift Libs`. To extend it, use `Eightshift_Libs\Helpers\Object_Helper` namespace.

Reviewer: ```suggestion
Object trait is located in `Eightshift Libs`. To extend it, use `Eightshift_Libs\Helpers\Object_Helper` class.
```

> All of us have some custom helpers and stuff that we use around on multiple the project. We have created this simple trait helper that you can use in any PHP class as a [trait](/eightshift-docs/docs/guides/extending-classes). To see all the class helper go [here](https://github.com/infinum/eightshift-libs/blob/develop/src/helpers/class-object-helper.php) and help your selves.

Reviewer 2: ```suggestion
All of us have some custom helpers and stuff that we use around on multiple the project. We have created this simple trait helper that you can use in any PHP class as a [trait](/eightshift-docs/docs/guides/extending-classes). To see all of the class helper go [here](https://github.com/infinum/eightshift-libs/blob/develop/src/helpers/class-object-helper.php).
```

> All of us have some custom helpers and stuff that we use around on multiple the project. We have created this simple trait helper that you can use in any PHP class as a [trait](/eightshift-docs/docs/guides/extending-classes). To see all the class helper go [here](https://github.com/infinum/eightshift-libs/blob/develop/src/helpers/class-object-helper.php) and help your selves.

Reviewer 3: ```suggestion
All of us have some custom helper functions that we use on multiple projects. We have created this simple trait helper that you can use in any PHP class as a [trait](/eightshift-docs/docs/guides/extending-classes). To see all the class helpers go [here](https://github.com/infinum/eightshift-libs/blob/develop/src/helpers/class-object-helper.php) and help yourselves.
```

> All of us have some custom helpers and stuff that we use around on multiple the project. We have created this simple trait helper that you can use in any PHP class as a [trait](/eightshift-docs/docs/guides/extending-classes). To see all the class helper go [here](https://github.com/infinum/eightshift-libs/blob/develop/src/helpers/class-object-helper.php) and help your selves.

Reviewer: ```suggestion
All of us have some custom helpers and stuff that we use around on multiple projects. We have created this simple helper that you can use in any PHP class as a [trait](/eightshift-docs/docs/guides/extending-classes). To see all the class helpers go [here](https://github.com/infinum/eightshift-libs/blob/develop/src/helpers/class-object-helper.php) and help your self.
```

## is_valid_xml

> Check if provided XML is a valid file. It uses a DOMDocument library.

Reviewer: ```suggestion
Check if the provided XML is valid. It uses a `DOMDocument` library.
```

* @param xml $xml Full xml document.
* @return boolean

## is_json

> Check if provided JSON is valid.

Reviewer: ```suggestion
Check if provided JSON is valid.

```

* @param string $string String to check.
* @return bool

## flatten_array

> Flattens multidimensional array.

Reviewer 3: ```suggestion
Flattens a multidimensional array.
```

> Flattens multidimensional array.

Reviewer: ```suggestion
Flattens a multidimensional array.
```

* @param  array $array Multidimensional array.
* @return array

## sanitize_array

> Sanitize all values in an array.

Reviewer: ```suggestion
Sanitize all the values in an array.
```
